webview-ui/src/manualTest/main.tsx

```diff
@@ -38,6 +38,7 @@ const contentTestScenarios: Record<ContentId, Scenario[]> = {
     kubectl: getKubectlScenarios(),
     aso: getASOScenarios(),
     tcpDump: getTCPDumpScenarios(),
+    retinaCapture: [],
```

Comment:
I think it's important to create the scenarios here to allow manual testing of the views. Doing this means you can launch the webview in a browser with hot reloading as you make code/styling changes. Without the ability to iterate quickly in this way it becomes very onerous to build more complex webviews.